client/src/components/DisplayAllCampaigns.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Flex } from '@radix-ui/themes';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { settlementApi } from '../service/api';
+import { TypeServerSettlement } from '../../../SettlementTypes';
+
+const DisplayAllCampaigns = () => {
+    const navigate = useNavigate();
+    const [allCampaigns, setAllCampaigns] = useState<TypeServerSettlement[]>([]);
+
+    useEffect(() => {
+        axios
+            .get(`${settlementApi}/get`)
+            .then((res) => {
+                console.log(res.data);
+                setAllCampaigns(res.data.settlements);
+            })
+            .catch((err) => console.log(err));
```

Comment:
You can use console.error() to log error messages fyi

Reply:
You should throw errors, don't hide them.  An app is better if it crashes when something fails, defensive programming just makes bugs go unnoticed, which makes them cause more pain than when they are in your face.

Reply:
good to know. updated